### ISSUE TITLE
"exec: Stdout already set" when staging a file

### ISSUE THREAD

**Issue by HannesOlszewski** [jesseduffield/lazygit#1741]
**Describe the bug**
Left lazygit open for a while so I don't know what exactly lead to this. After some coding I wanted to stage my files with `space` and the error "exec: Stdout already set" was shown. No file was staged. Restarting lazygit did not help and I needed to delete `.git/index.lock` to resolve this.

**To Reproduce**
1. Stage file with `space`
2. See error

**Expected behavior**
The file being staged and no error shown.

**Screenshots**
<img width="1920" alt="Screenshot 2022-01-28 at 22 06 00" src="https://user-images.githubusercontent.com/13519957/151622565-974bd37c-f1e9-4540-8dfe-670e19b1b762.png">

**Desktop (please complete the following information):**
 - OS: MacOS 12.2 (M1)
 - Lazygit Version: v0.32.2
 - The last commit id if you built project from sources: Not applicable, installed through Homebrew

**Comment by jesseduffield** [comment]
On master I've fixed the issue with the error not being very informative, however the index.lock bug remains. Haven't been able to determine what's causing it. Any help is much appreciated!

**Comment by simonvizzini** [comment]
I just ran into this issue (exec: Stdout already set), and the last thing I did in lazygit before it happened was to rebase my working branch with unstaged changes which resulted in some merge conflicts. What was strange is that this time I didn't get the usual "Merge conflicts! Resolve or press enter to abort" message (don't remember the exact spelling). The merge conflicts were not critical and so I simply reverted the files that contained the conflicts (via lazygits 'd' shortcut). After that I continued to work on my code and when I came back to lazygit (which I always keep open in a terminal window) I couldn't stage files anymore, getting the mentioned "Stdout already exists" error. Not sure if these steps somehow triggered the error, but I hope it helps in identifying the root cause. I'll keep observing it and if it happens again I'll try to provide more details.
